Case Study/vehicalDekho/helper/VehicleHelper.java

```diff
@@ -0,0 +1,54 @@
+package helper;
+
+import model.Bike;
+import model.Car;
+import model.Vehicle;
+
+public class VehicleHelper {
```

Comment:
Although it is not mentioned in the problem statement that the user should NOT be allowed to instantiate "VehicleHelper" though 'new' operator. E.g., new VehicleHelper() should not be allowed as it is a helper/utility class only.